Testscripts/Linux/CPUOfflineKernelBuild.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+# This script will execute the CPU channel change along with vmbus interrupt re-assignment
+# This feature will be enabled the kernel version 5.7+
+########################################################################################################
+# Source utils.sh
+. utils.sh || {
+	echo "Error: unable to source utils.sh!"
+	echo "TestAborted" > state.txt
+	exit 0
+}
+# Source constants file and initialize most common variables
+UtilsInit
+
+# Constants/Globals
+# Get distro information
+GetDistro
+
+function Main() {
+
+	basedir=$(pwd)
```

Comment:
I think we use 4 space for next line indent ?

Reply:
It looks ok to me in the local file. I think this is Github UI thing